USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs

```diff
@@ -17,6 +17,7 @@ public class HtmlRenderer
         private IList<string> TOCEntries;
         private CMarker CurrentChapter;
         private VMarker CurrentVerse;
+        private int NextFootnoteUniqueID = 1;

```

Comment:
This might be unexpected behavior because this doesn't get set every time rendering is called. So if render is called multiple times it will end up just continuing upwards instead of resetting.